Лабораторная 1/task2.py

```diff
@@ -0,0 +1,5 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+
+k=4*100*50*25
+inf=1.44*1024*1024
```

Comment:
Вы молодец, задача решена!
Но позволю себе немного уточнить.
Сейчас у вас много "магических" чисел, которые не привязаны к переменным.
Было бы лучше, если бы вы создали несколько переменных с понятным человеку названием.
Например вот так:
```suggestion
disk_volume = 1.44 # информационный объем дискеты
number_of_pages = 100 # количество страниц в книге
number_of_str = 50 # число строк на странице
number_of_symbols = 25 # количество символов в стркое
one_volume_of_symbol = 4 # количество байтов для одного символа
```
А затем посчитали количество книг следующим образом:
`book_volume = disk_volume * 1024 * 1024 // (number_of_pages * number_of_str * number_of_symbols * one_volume_of_symbol)`